### ISSUE TITLE
Liens présents dans la page PCTRun

### ISSUE THREAD

**Issue by gquerret** [Riverside-Software/pct-wiki#5]
Sur [cette page](https://github.com/Riverside-Software/pct/wiki/PCTRun#parameters-as-nested-elements), le premier lien (DbConnection or PCTConnection) ne fonctionne pas dans le wiki, mais fonctionne dans le site publié.
Le troisième lien (Option or PCTRunOption) fonctionne dans le wiki mais pas dans le site publié.

Cela vient de la syntaxe `[[ ]]` ( je crois), il faut une solution qui fonctionne dans les deux cas.

**Comment by BastienThoral** [comment]
@gquerret La syntaxe est bonne dans pct-wiki, le lien fonctionnedans le wiki est dans les pages générés, tu veux que je modifie les liens du site "principal" PCT ?

**Comment by gquerret** [comment]
Non, si tu cliques sur le lien tu ne tombes pas sur la même page. Dans le premier cas c'est le texte brut, et dans le deuxième c'est la page avec le rendu markdown.
